File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/Model/Logs/CommonLog.java
package com.siliconlabs.bledemo.Browser.Model.Logs;

import com.siliconlabs.bledemo.Browser.Model.LogType;

public class CommonLog extends Log {

    public CommonLog(String value, String deviceAddress) {
        setLogTime(getTime());
        setLogInfo(value);
        setLogType(LogType.INFO); //malo wazne
        setDeviceAddress(deviceAddress);
    }
}

File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/Models/Logs/CommonLog.kt
package com.siliconlabs.bledemo.Browser.Models.Logs

import com.siliconlabs.bledemo.Browser.Models.LogType

class CommonLog(value: String, deviceAddress: String) : Log() {
    init {
        logTime = getTime()
        logInfo = value
        logType = LogType.INFO
        super.deviceAddress = deviceAddress
    }
}
